src/us/kbase/auth2/service/AuthBuilder.java

```diff
@@ -62,15 +64,17 @@ public AuthBuilder(
 	
 	private MongoClient buildMongo(final AuthStartupConfig c) throws StorageInitException {
 		//TODO ZLATER MONGO handle shards & replica sets
+		MongoClientSettings.Builder mongoBuilder = MongoClientSettings.builder().applyToClusterSettings(
```

Comment:
```suggestion
		final MongoClientSettings.Builder mongoBuilder = MongoClientSettings.builder().applyToClusterSettings(
```

Reply:
is `final` here necessary? To avoid mongoBuilder getting reassigned?

Reply:
Necessary? No. Good practice? Yes, prevents bugs by accidentally assigning variables (which isn't likely here) and also allows the compiler to perform optimizations where it knows that variables are never going to be reassigned